man/memkind.3

```diff
@@ -258,15 +258,20 @@ on a call to
 .BR memkind_pmem (3)).
 This allows to create a pmem memkind of a pretty large size without the
 need to reserve in advance the corresponding file system space for the entire
-heap.  The minimum
+heap. If the value of
 .I max_size
-value allowed by the library is defined in
+is equal 0, pmem memkind is not limited by the file system specified by the
```

Comment:
pmem memkind is only limited by the capacity of the file system mounted under *dir*.

Reply:
Done.